torch/nn/quantized/modules/conv.py

```diff
@@ -38,10 +38,19 @@ def __init__(self, in_channels, out_channels, kernel_size, stride=1,
         self.output_padding = 0
         self.groups = groups
         self.padding_mode = padding_mode
+
+        # Although not necessary for PTQ, for debugging purposes set the scale
+        # to be able to represent real defaults.
```

Comment:
what about weights? we have empty weights as default right now

Reply:
I don't think wee need to set the weights. This change is for debugging, and in practice the weights will be replaced, right? If you think this is necessary, I will set the weights to random.